content/terms_and_definitions/terms_and_definitions.adoc

```diff
@@ -6,7 +6,86 @@ ifndef::include-only-once[]
 include::{root-path}_config.adoc[]
 endif::[]
 
-TODO: Add terms and definitions
+3D Asset::
+A 3D model. containing the 3D geometry, combined with assigned material properties and specific meta data.
 
-Static road network::
-Collection of connected roads enriched by static objects that do not change during runtime of a simulation.
+3D Model::
+A (mathematical) geometrical, coordinate-based representation of the surface of an object.
```

Comment:
A mathematical coordinate-based representation of an object's geometrical surface.

Reply:
General suggestion for 3D model/asset/environment: Maybe only list 3D model and then list asset and environment as related terms to better show how these concepts belong together?

Reply:
Yes, referencing the definitions of asset and environment is a good idea!

Reply:
updated definition

TODO: add relations